js/index.js

```diff
@@ -1,273 +0,0 @@
-let windowW = window.innerWidth;
-let windowH = window.innerHeight;
-let bgColor = "lightgray";
-let jack;
-let enemies;
-let horizontalLine;
-let lineGap = windowH / 8;
-let holeW = lineGap;
-let jumpingTime;
-
-const ironHackBlue = "#009bff";
-
-let gameSpeed = 4;
-let score;
-let gameTime;
-let bonusTime = 10;
-
-let spaceBarKey = 32;
-let gameState = "playing";
-
-let bgMusic;
-let jumpSound;
-let winSound;
-let loseSound;
-let moveSound;
-let gameMusic = "OFF";
-
-function preload() {
-  jackImg = loadImage("assets/images/jhack.png");
-  jackImgLeft = loadImage("assets/images/jhack_left.png");
-  jackImgRight = loadImage("assets/images/jhack_right.png");
-  gameMusicImg = loadImage("assets/images/extra.png");
-  bgMusic = loadSound("assets/sounds/bgMusic.mp3");
-  jumpSound = loadSound("assets/sounds/jump.mp3");
-  fallSound = loadSound("assets/sounds/fall1.mp3");
-  winSound = loadSound("assets/sounds/win.wav");
-  loseSound = loadSound("assets/sounds/lose.wav");
-  moveSound = loadSound("assets/sounds/move.mp3");
-  fallSound = loadSound("assets/sounds/fall1.mp3");
-}
-
-function setup() {
-  const canvas = createCanvas(windowW, windowH);
-  canvas.parent("game-board");
-  bgMusic.setVolume(0.2);
-  initializeGame();
-  soundFormats("mp3", "wav");
-  noLoop();
-}
-
-function draw() {
-  if (gameState === "playing") {
-    playGame();
-    return;
-  }
-  if (gameState === "gameover") {
-    gameOver();
-  }
-  if (gameState === "win") {
-    win();
-  }
-}
-function getHolesDirectlyAbove() {
-  return holes.filter((hole) => {
-    const dif = jack.y - hole.y;
-    if (dif < 0 || dif > lineGap) {
-      return false;
-    }
-    return true;
-  });
-}
-
-// TRYING TO GET HOLES BELOW //
-
-function getHolesDirectlyBelow() {
-  return holes.filter((hole) => {
-    if (jack.y + jack.h === hole.y) {
-      return true;
-    }
-    return false;
-  });
-}
-
-function collidesHoleBelow() {
-  const holesDirectyBelowUs = getHolesDirectlyBelow();
-  const collidingHoleBelow = holesDirectyBelowUs.find((hole) => {
-    return collisionDetection(hole, jack);
-  });
-  if (collidingHoleBelow && new Date() - jumpingTime > 600) {
-    jack.y += lineGap;
-    fallSound.setVolume(0.3);
-    fallSound.play();
-  }
-}
-
-function keyPressed() {
-  if (keyCode === spaceBarKey || keyCode === UP_ARROW) {
-    const holesDirectyAboveUs = getHolesDirectlyAbove();
-    const collidingHole = holesDirectyAboveUs.find((hole) => {
-      return collisionDetection(hole, jack);
-    });
-    if (collidingHole) {
-      jack.jump();
-      jumpingTime = new Date();
-      holes.push(
-        new Hole(random(0, windowW), floor(random(9)) * lineGap, "right"),
```

Comment:
you might consider moving this into the constructor.

and then checking for "optional" overwrite arguments. 

i.e. in hole.js
```js
class Hole {
   constructor({x = random(0, windowW), y = floor(random(9)) * lineGap, direction = "right" } = {}) {
      this.x = x
      this.y = y
      this.direction = direction
   }
}
```

then you could call it like so: 
```js
new Hole( {direction: "right" })
new Hole( {direction: "left" })
```
or even like so: 
```js 
new Hole()
```